docs/concepts/extension_points.rst

```diff
@@ -105,45 +103,66 @@ Here are the different integration points that python plugins can use:
      - Status
      - Details
    * - Django App Plugin (``lms.djangoapp`` and ``cms.djangoapp``)
-     - Adopt, Stable
+     - ⚠️ **Provisional**
      - A "Django app plugin" is a self-contained Django `Application`_ that can define models (MySQL tables), new REST APIs, signal listeners, asynchronous tasks, and more. Even some parts of the core platform are implemented as Django app plugins, for better separation of concerns (``announcements``, ``credentials``, ``grades``, etc.) Read the `Django app plugin documentation`_ to learn more.
 
        Plugins can also inject custom data into django template contexts, to affect standard pages delivered by the core platform. See `Plugin Contexts`_ to learn more.
    * - Course tab (``openedx.course_tab``)
-     - Hold, Stable
+     - 🕰️ **Legacy**
      - A course tab plugin adds a new tab shown to learners within a course. ``courseware``, ``course_info``, and ``discussion`` are examples of built-in tab plugins. Read the `course tabs documentation`_ to learn more.
 
        This API may be changing soon with the new Courseware microfrontend implementation.
    * - Course tool (``openedx.course_tool``)
-     - Hold, Stable
+     - 🕰️ **Legacy**
      - The course home page (the landing page for the course) includes a "Course Tools" section that provides links to "tools" associated with the course. Examples of course tool plugins included in the core are reviews, updates, and bookmarks. See |course_tools.py|_ to learn more.
 
        This API may be changing soon with the new Courseware microfrontend implementation.
    * - Custom registration form app (``REGISTRATION_EXTENSION_FORM`` Django setting in the LMS)
-     - Trial, Stable
+     - ⚠️ **Provisional**
```

Comment:
MIT doesn't use this, but based on traffic on https://discuss.openedx.org/, I'm surprised to see this go from stable to provisional.